src/docs/development/data-and-backend/google-apis.md

```diff
@@ -126,7 +126,7 @@ When event value is not `null`, you can create an authenticated client.
 
 <?code-excerpt "lib/main.dart" skip="124" take="1"?>
 ```dart
-var httpClient = (await _googleSignIn.authenticatedClient())!;
+];
```

Comment:
I suspect this is incorrect. I'm guessing that it should really be using comments in the source to mark out the part of the codebase to pull.

FYI @kevmoo

Reply:
Great catch. I've added docregions to the code to correct this.